serverless/functions/reference-content/functions-billing.mdx

```diff
@@ -0,0 +1,22 @@
+---
+meta:
+  title: Serverless Functions billing
+  description: This section explains how users are billed for Serverless Functions
+content:
+  h1: Serverless Functions billing
+  paragraph: This section explains how users are billed for Serverless Functions
+tags: functions serverless serverless-functions billing serverless-billing serverless-functions-billing
+dates:
+  validation: 2023-06-15
+  posted: 2023-06-15
+categories:
+  - serverless
+---
+
+Serverless Functions are billed on a pay-as-you-go basis depending on: 
+
+* Monthly Request number
+* Resource consumption
+* Resources provision
```

Comment:
```suggestion
* **Monthly Request number:** each time your function is invoked we increase a counter.
* **Resource consumption:** this component is obtained by multiplying the memory tiers chosen (128MB, 256MB, 512MB, 1024MB, 2048MB) by the duration of each function invocation.
* **Resources provision:** in order to mitigate cold start, users can choose to keep some function Instances warm (by filing the min scale value). We then charge for the provisioned resources similarly to the Resources consumption component.

The scheme below illustrates our billing model:

<Lightbox src={["scaleway-Billing-FaaS-1-DEF.webp", "scaleway-Billing-FaaS-2-DEF.webp"]} size={"medium"}  alt={["Functions Billing", "Functions Billing"]} />
```

Reply:
What do you think about taking more content from the FAQ, IMHO customers expect to find at least some detailed information right away, it is painful to have to click on another link a leave the current section of the documentation to get some info

Reply:
Ok.

Reply:
As discussed, Billing stays in FAQ, with a link from Ref content to it.